### PR TITLE
U4-8446 - Navigational map for the grid

### PR DESCRIPTION
**Pull request by lars-erik**
Adds a navigational widget to the grid.

### CODE REVIEW

**Comment by lars-erik** [comment]
It needs a bit of CSS love from some guru.
Comments in the code.

File: src/Umbraco.Web.UI.Client/src/less/gridview.less [hunk 1]
```diff
@@ -8,7 +8,7 @@
 .usky-grid IFRAME {overflow:hidden;}
 
 
- // Sortabel
+ // Sortable
```
Comment (lars-erik) [inline src/Umbraco.Web.UI.Client/src/less/gridview.less]: Couldn't help myself. :)

File: src/Umbraco.Web.UI.Client/src/less/gridview.less [hunk 2]
```diff
@@ -895,3 +895,83 @@
 }
 
 .usky-grid-configuration a.uSky-templates-column{height: 70px !important;}
+
+// Nav
+
+.umb-grid-nav {
```
Comment (lars-erik) [inline src/Umbraco.Web.UI.Client/src/less/gridview.less]: Should possibly have its own .less?

File: src/Umbraco.Web.UI.Client/src/less/gridview.less [hunk 3]
```diff
@@ -895,3 +895,83 @@
...
+    .column() {
+        display: inline-block;
+        float: left;
+    }
+
+    background-color: white;
+    border: 1px solid gray;
+    opacity: .8;
+    position: absolute;
```
Comment (lars-erik) [inline src/Umbraco.Web.UI.Client/src/less/gridview.less]: Position looks messed up when the sticy meets the non-sticky.

File: src/Umbraco.Web.UI.Client/src/views/propertyeditors/grid/grid.html [hunk 4]
```diff
@@ -1,6 +1,7 @@
 <div ng-controller="Umbraco.PropertyEditors.GridController" class="umb-grid umb-editor clearfix" id="umb-grid">
 
     <umb-editor-sub-header>
+        <label style="float: right;"><input type="checkbox" ng-model="showNav" />Nav</label>
```
Comment (lars-erik) [inline src/Umbraco.Web.UI.Client/src/views/propertyeditors/grid/grid.html]: So sorry again. The sub-header-content-right isolates scope, so can't use it.
Float right doesn't work for some reason. CSS love please. ;)

**Comment by lars-erik** [comment]
whoop - and a test: https://gist.github.com/lars-erik/5816d21a9b860bda612dfb6c1b2e33fa

**Comment by readingdancer** [comment]
Hi @lars-erik 

I noticed this pull request is currently the oldest open pull request, it's also 6000+ commits behind the dev-v7 branch. Could you update it and I'd be happy to take a look at the CSS issues you were having, assuming it's still a relevant pull request :)

Cheers, Chris

**Comment by lars-erik** [comment]
Hey and thanks, Chris,

Since I got the grid events merged this can now be a package.
I'll close the PR and someday make a package from it.

Thanks again. :)

**Pull request closed without merging**